DalTube/DalTube/WelcomeViewController.swift

```diff
@@ -15,12 +15,26 @@ class WelcomeViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         setNameInLabel()
-        // Do any additional setup after loading the view.
+        
+    }
+    
+    //확인
+    @IBAction func goTabBarVC(_ sender: Any) {
+        guard let tabBarVC = self.storyboard?.instantiateViewController(withIdentifier: "CustomTabBarController") else {return}
+        
+        tabBarVC.modalPresentationStyle = .fullScreen
+        self.present(tabBarVC, animated: true, completion: nil)
+    }
+    
+    
+    //다른 계정으로 로그인하기
+    @IBAction func touchUpToGoToLoginView(_ sender: Any) {
+        self.navigationController?.popToRootViewController(animated: true)
```

Comment:
```suggestion
       guard let pvc = presentingViewController as? UINavigationController else { return }
       
      dismiss(animated: true) {
                pvc.popToRootViewController(animated: false)
      }
```

이런 식으로 사용하면 present를 사용한 뷰에서도 로그인뷰로 돌아갈 수 있습니다!

`presentingViewController`는 현재 뷰를 띄운 뷰를 말하는데, 지금 우리가 하는 프로젝트로 치면 로그인뷰와 회원가입뷰를 가지고 있는 NavigationController를 뜻하겠죠? 

그 presentingViewController를 가지고 와서 `completion`에서  `popToRootViewController`를 해주면 dismiss될 때 현재 뷰뿐만아니라 현재 뷰를 띄워준 presentingViewController도 rootView로 pop되게 됩니다! 
completion은 dismiss를 실행하고 나서 해당 코드를 실행하기 때문에 dismiss 후에 pop시켜줍니다!